llvm/include/llvm/MC/MCTargetOptions.h

```diff
@@ -61,6 +61,9 @@ class MCTargetOptions {
 
   bool Dwarf64 : 1;
 
+  // Use CREL relocation format for ELF.
+  bool Crel = false;
```

Comment:
Nit here and elsewhere: Is "Crel" a correct way of spelling this? Should it be "CRel" (for "CompressedRelocations")? I'm assuming you're deliberately avoiding using CREL in this context, but an argument could be made for that too.

Reply:
If LLVM adopts https://llvm.org/docs/Proposals/VariableNames.html , I'd like to use `crel` instead of `cRel`. But with the capitalized naming, I'd prefer `Crel`. This makes many functions' names `xxxCrels` align better with `xxxRels`.